pkg/utils/utils.go

```diff
@@ -217,6 +218,14 @@ func IsEqual(key string, value interface{}, reqValue interface{}) (bool, error)
 	}
 }
 
+// ToUpperFirst will change the first letter of string to uppercase
```

Comment:
Could strings.Title meet your requirements？

Reply:
thanks, I change it to strings.Title, done